levechad/settings.py

```diff
@@ -151,10 +152,34 @@
 USE_TZ = True
 
 
+LOGIN_REDIRECT_URL = '/server'
+
+
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
-
 STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
-STATIC_URL = "/static/"
-
-LOGIN_REDIRECT_URL = '/server'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'mediafiles')
+
+if not DEBUG:
```

Comment:
let's be consistent with the rest of the page
```suggestion
if ENV != 'PRODUCTION':
```